libs/api/domains/housing-benefits/src/lib/housing-benefits.resolver.ts

```diff
@@ -0,0 +1,40 @@
+import { Query, Resolver, Args } from '@nestjs/graphql'
+import { UseGuards } from '@nestjs/common'
+
+import { ApiScope } from '@island.is/auth/scopes'
+import type { User } from '@island.is/auth-nest-tools'
+import {
+  IdsUserGuard,
+  ScopesGuard,
+  CurrentUser,
+  Scopes,
+} from '@island.is/auth-nest-tools'
+import { HmsHousingBenefitsClientService } from '@island.is/clients/hms-housing-benefits'
+import { Audit } from '@island.is/nest/audit'
+import { HousingBenefitPaymentsResponse } from './models/housingBenefit.model'
+import { HousingBenefitPaymentsInput } from './dto/getHmsLoansPaymenthistory.input'
+
+@UseGuards(IdsUserGuard, ScopesGuard)
+@Scopes(ApiScope.financeOverview)
+@Resolver()
+@Audit({ namespace: '@island.is/api/hms-housing-benefits' })
+export class HousingBenefitsResolver {
+  constructor(
+    private housingBenefitsService: HmsHousingBenefitsClientService,
+  ) {}
+
+  @Query(() => HousingBenefitPaymentsResponse, {
+    name: 'housingBenefitPayments',
```

Comment:
~ Minor details but missing 's' at the end to match exact api domain name 😅. And I noticed now that it is missing in the type names as well.
```suggestion
    name: 'housingBenefitsPayments',
```